app.rb

```diff
@@ -1,8 +1,21 @@
 require 'sinatra/base'
+require './lib/peeps'
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+  get '/' do
+    'Welcome to Chitter!'
+  end
```

Comment:
From a user journey perspective, could it be worth adding some functionality to the homepage? Just so people know to go to /peeps? Or maybe all the functionality of /peeps could sit on the homepage instead?